helpdesk_mgmt_timesheet/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,11 @@
+To configure this module, you need to:
+#. Allow Timesheet for a Helpdesk's Team
```

Comment:
Need empty line between for proper RST syntax

Reply:
Fixed, thanks a lot.